nodes/clusters/nodes-cluster-overcommit.adoc

```diff
@@ -100,15 +94,21 @@ include::modules/nodes-cluster-overcommit-node-resources.adoc[leveloffset=+2]
 
 include::modules/nodes-cluster-overcommit-node-disable.adoc[leveloffset=+2]
 
+endif::openshift-rosa,openshift-rosa-classic,openshift-rosa-hcp,openshift-dedicated[]
+
+//ROSA and Dedicated can only do the following, so no ifndef on these
 include::modules/nodes-cluster-project-overcommit.adoc[leveloffset=+1]
 
 include::modules/nodes-cluster-overcommit-project-disable.adoc[leveloffset=+2]
 
-ifndef::openshift-rosa,openshift-dedicated[]
 [role="_additional-resources"]
 [id="nodes-cluster-overcommit-addtl-resources"]
 == Additional resources
-
+ifndef::openshift-rosa,openshift-rosa-classic,openshift-rosa-hcp,openshift-dedicated[]
 * xref:../../applications/deployments/managing-deployment-processes.adoc#deployments-triggers_deployment-operations[Setting deployment resources].
 * xref:../../nodes/nodes/nodes-nodes-resources-configuring.adoc#nodes-nodes-resources-configuring-setting_nodes-nodes-resources-configuring[Allocating resources for nodes].
```

Comment:
```suggestion
* xref:../../applications/deployments/managing-deployment-processes.adoc#deployments-triggers_deployment-operations[Setting deployment resources]
* xref:../../nodes/nodes/nodes-nodes-resources-configuring.adoc#nodes-nodes-resources-configuring-setting_nodes-nodes-resources-configuring[Allocating resources for nodes]
```
Since this is a part of this PR - if possible, can you please consider removing these periods per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-xref-attribute?